qiskit_experiments/calibration/management/backend_calibrations.py

```diff
@@ -147,18 +148,35 @@ def get_meas_frequencies(
         """
         return self._get_frequencies(FrequencyElement.READOUT, group, cutoff_date)
 
-    def export_backend(self) -> Backend:
+    def export_backend(
+            self,
+            basis_gates: Optional[List[str]] = None,
+            group: str = "default",
+            cutoff_date: Optional[datetime] = None,
+    ) -> Backend:
```

Comment:
One more thing. With this methodology we export the entire cals at every experiment for the sake of transpiling. For a small backend this is not an issue but it becomes very inefficient as we have more and more gates and qubits. Would it be possible to do a partial export by restricting to certain schedules and qubits?

Reply:
I think this it's possible, but how user can know which basis schedules are used before transpiling (or unrolling) the circuit? If some gate is missing in the list the execute will just use backend cal so it can run anyways. But this means end users have responsibility to manage the cal entries. For example, if one wants to use optimization level > 2 and stochastic mapper, it is hard to predict qubit set.

Reply:
Perhaps allowing `qubits` option to (optionally) limit qubit set to export is still efficient.

Reply:
See https://github.com/Qiskit/qiskit-experiments/pull/185/commits/7f9613edbf485a100130853d3d2ba6e90fde863a